client/state/comments/selectors/get-comment-by-id.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import { filter, find, flatMap } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { deconstructStateKey, getErrorKey } from 'state/comments/utils';
+
+import 'state/comments/init';
+
+export function getCommentById( { state, commentId, siteId } ) {
+	const errorKey = getErrorKey( siteId, commentId );
+	if ( ( state.comments.errors ?? {} )[ errorKey ] ) {
```

Comment:
`errors`'s initial state is `{}`; so is the nullish coalescing necessary here?

Reply:
It starts off as an object, and it doesn't appear to be possible to get changed into something else. Good catch! Removing the check.